tests/general/InvalidLinkBearTest.py

```diff
@@ -139,11 +166,44 @@ def test_run(self):
         http://httpbin.org/status/503
         http://www.notexample.com/404
         http://exampe.com/404
-        http://example.co.in/404"""
+        http://example.co.in/404
+        <ruleset name="test" xmlns="http://this.isa.namespace/ruleset/7.0.0"
+        xmlns:xsi="http://this.is.another/kindof/namespace"
+        xsi:schemaLocation="http://this.namespace.dosent/exists/7.0.0"
+        xsi:schemaLocation="http://httpbin.com/404">"""
 
         for line in invalid_file.splitlines():
             self.assertResult(invalid_file=[line])
 
+    def test_severity(self):
+        info_severity_file = '''
+        <ruleset name="test" xmlns="http://this.isa.namespace/ruleset/7.0.0"
+        xmlns:xsi="http://this.is.another/kindof/namespace">
+        <ruleset name="test" xmlns="http://this.isa.namespace/ruleset/7.0.0"
+        xmlns:xsi="http://this.isa.namespace/ruleset/7.0.0"
+        xsi:schemaLocation="http://this.isa.namespace/ruleset/7.0.0">
+        '''.splitlines()
+
+        self.assertSeverity(info_severity_file, RESULT_SEVERITY.INFO)
+
+        normal_severity_file = '''
+        http://httpbin.org/status/404
+        http://httpbin.org/status/410
+        http://httpbin.org/status/500
+        http://httpbin.org/status/503
+        http://httpbin.org/status/301
+        http://httpbin.org/status/302
+        xsi:schemaLocation="http://httpbin.com/404"
+        '''.splitlines()
+
+        self.assertSeverity(normal_severity_file, RESULT_SEVERITY.NORMAL)
+
+        major_severity_file = '''
+        http://coalaisthebest.com
```

Comment:
Using `http://this.isa.namespace/` , and putting it at the top of this test method, would help the reader understand.